CONTRIBUTING.md

```diff
@@ -513,5 +513,18 @@ To install the API to your local maven repository, do the following:
 You can now use the API in your plugin to test it before PRing. You will also
 need to do this to build the Server with the implemented API.
 
+### How can I use a debugger on the Paper Server?
+
+To debug the paper server, do the following:
+
+- Go through the normal steps for debugging a plugin ([Intellij](https://www.spigotmc.org/wiki/intellij-debug-your-plugin/) or [Eclipse](https://www.spigotmc.org/wiki/eclipse-debug-your-plugin/)).
```

Comment:
why would we recommend a remote configuration over a normal jar configuration?

Reply:
Are there practical differences? I just wanted to link the basic instructions so they wouldn't have to be typed out here.